fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -160,7 +231,8 @@ protected void initializeIndexSearcher() {
      * @param firstTimeThrough  If true, will write an empty index and will then re-open the searcher
      */
     private void reopenIndexSearcher(boolean firstTimeThrough) {
-        lock.writeLock().lock();
+//        lock.writeLock().lock();
```

Comment:
Just delete this line. Commented code is noisy and unnecessary in the era of version control.

Reply:
sorry, forgot to delete unused comments -.-